pkg/config/config.go

```diff
@@ -97,6 +97,9 @@ type RTCConfig struct {
 	// allow TCP and TURN/TLS fallback
 	AllowTCPFallback *bool `yaml:"allow_tcp_fallback,omitempty"`
 
+	// check RTT for TCP fallback, will skip tcp ice if signaling RTT greater than tcpGoodRTT(80ms)
+	TCPFallbackCheckRTT *bool `yaml:"tcp_fallback_check_rtt,omitempty"`
```

Comment:
instead of a bool value in the global configurations, I think it'd be more flexible we could pass in a `TCPFallbackMaxRTT` to NewParticipant during construction.

It would be more flexible to depending on the project/participant. By default (0 for unset), it would allow tcp fallback.

Reply:
the config.yaml file should be treated as an user-facing API.. things added to this file would be difficult to remove as users start depending on them.